### ISSUE TITLE
example 6.3 screenshot

### ISSUE THREAD

**Issue by shiffman** [shiffman/LearningProcessing#22]
The book I'm referring to is "Learning Processing" A Beginner's Guide to Programming Images, Animation, and Interaction. In  lesson 2, chapter 6, example 6.3 the code is correct but the illustration is not. It looks like it illustrates:
 size(200,200);
background(255);
int y = 60;
int x = 50;
int spacing = 10;
int len = 20;
int endLegs = 150;
stroke(0);
while (x <= endLegs) {
 line (x,y,x, y + len);
x = x + spacing;
}

rather than

 size(200,200);
background(255);
int y = 80;
int x = 50;
int spacing = 10;
int len = 20;
int endLegs = 150;
stroke(0);
while (x <= endLegs) {
 line (x,y,x, y + len);
x = x + spacing;
}

In other words, int y = 80 should be int y = 60 to match the illustration (fig 6.3).

**Comment by shiffman** [comment]
https://github.com/shiffman/Learning-Processing-2.0/issues/12 needs to be resolved first
